### PR TITLE
adapt for olap writeback

### PR DESCRIPTION
**Pull request by zhoney**
Change-Id: I86f9cb0307f4dd615b9672667e56bb906f9086a7

### CODE REVIEW

File: src/main/java/com/baidu/hugegraph/driver/SchemaManager.java [hunk 1]
```diff
@@ -87,8 +102,36 @@ public PropertyKey eliminatePropertyKey(PropertyKey propertyKey) {
         return this.propertyKeyAPI.eliminate(propertyKey);
     }
 
+    public PropertyKey clearPropertyKey(PropertyKey propertyKey) {
+        return this.clearPropertyKey(propertyKey, TASK_TIMEOUT);
+    }
+
+    public PropertyKey clearPropertyKey(PropertyKey propertyKey, long seconds) {
+        PropertyKey.PropertyKeyWithTask pkwt = this.propertyKeyAPI
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/driver/SchemaManager.java:110]: rename to task

File: src/main/java/com/baidu/hugegraph/driver/SchemaManager.java [hunk 2]
```diff
@@ -87,8 +102,36 @@ public PropertyKey eliminatePropertyKey(PropertyKey propertyKey) {
...
+                                                   .clear(propertyKey);
+        if (pkwt.taskId() != 0L) {
+            this.taskAPI.waitUntilTaskSuccess(pkwt.taskId(), seconds);
+        }
+        return pkwt.propertyKey();
+    }
+
+    public long clearPropertyKeyAsync(PropertyKey propertyKey) {
+        PropertyKey.PropertyKeyWithTask pkwt = this.propertyKeyAPI
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/driver/SchemaManager.java:119]: ditto

File: src/main/java/com/baidu/hugegraph/structure/constant/ReadFrequency.java [hunk 3]
```diff
@@ -41,4 +45,14 @@ public byte code() {
...
         return this.name;
     }
+
+    public boolean oltp() {
+        return this == OLTP;
+    }
+
+    public boolean olap() {
+        return this == OLAP_NONE ||
+                this == OLAP_RANGE ||
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/structure/constant/ReadFrequency.java:55]: align

File: src/main/java/com/baidu/hugegraph/structure/constant/ReadFrequency.java [hunk 4]
```diff
@@ -23,7 +23,11 @@
 
     OLTP(1, "oltp"),
 
-    OLAP(2, "olap");
+    OLAP_NONE(2, "olap_none"),
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/structure/constant/ReadFrequency.java:26]: OLAP_COMMON?

File: src/test/java/com/baidu/hugegraph/api/BaseApiTest.java [hunk 5]
```diff
@@ -145,10 +145,16 @@ protected static void clearData() {
...
 
     protected static void waitUntilTaskCompleted(long taskId) {
+        if (taskId == 0) {
+            return;
+        }
         taskAPI.waitUntilTaskSuccess(taskId, TIMEOUT);
     }
 
     protected static void waitUntilTaskCompleted(long taskId, long timeout) {
+        if (taskId == 0) {
```
Comment (javeme) [inline src/test/java/com/baidu/hugegraph/api/BaseApiTest.java:155]: 0L

File: src/main/java/com/baidu/hugegraph/structure/constant/ReadFrequency.java [hunk 6]
```diff
@@ -23,7 +23,11 @@
...
+
+    OLAP_SECONDARY(3, "olap_secondary"),
+
+    OLAP_RANGE(4, "olap_range");
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/structure/constant/ReadFrequency.java:30]: not sure when it need to pass OLAP_SECONDARY or OLAP_RANGE to server

Comment (zhoney) [reply src/main/java/com/baidu/hugegraph/structure/constant/ReadFrequency.java:30]: add comments

**Review by javeme — APPROVED** [review]

**Review by Linary — APPROVED** [review]

**Pull request merged**